lib/components/viewers/realtime-status-label.tsx

```diff
@@ -106,12 +107,12 @@ const RealtimeStatusLabel = ({
         <TimeStruck aria-hidden>
           <FormattedTime timeStyle="short" value={originalTime} />
         </TimeStruck>
-        <div>
+        <div className={isRealtime ? 'realtime-icon realtime' : ''}>
```

Comment:
Adding the realtime class is adding the realtime icon. I think this is fine, but we need to account for the new positioning.
<img width="407" alt="Screenshot 2025-06-26 at 10 57 15 AM" src="https://github.com/user-attachments/assets/b5752168-4a5a-4b60-b364-a17e940cf6c0" />

Otherwise, we should avoid using the `realtime` class